db/Pos/DB/Class.hs

```diff
@@ -185,9 +185,11 @@ gsUnlockStakeEpoch = bvdUnlockStakeEpoch <$> gsAdoptedBVData
 
 -- | Checks if provided epoch is in the bootstrap era.
 gsIsBootstrapEra :: MonadGState m => EpochIndex -> m Bool
-gsIsBootstrapEra epoch = do
-    unlockStakeEpoch <- gsUnlockStakeEpoch
-    return $ isBootstrapEra unlockStakeEpoch epoch
+gsIsBootstrapEra epoch =
+    if isDevelopment then pure False
```

Comment:
this code from CSM-322 and CSM-359, I propose to do it @flyingleafe 
I want to merge this PR ASAP